ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/resources/StreamedQueryResourceTest.java

```diff
@@ -131,6 +133,32 @@ public void shouldReturn400OnBuildMultipleQueriesError() throws Exception {
         errorMessage.getMessage(), containsString("some msg only the engine would use"));
   }
 
+  @Test
+  public void shouldFailIfIsNotAcceptingStatements() throws Exception {
+    // Given:
+    final String queryString = "SELECT * FROM test_stream;";
+    final KsqlConfig ksqlConfig = mock(KsqlConfig.class);
```

Comment:
Move mock creation, creation of thing under test, and common behaviour into setup method.